python/lsst/dax/obscore/obscore_exporter.py

```diff
@@ -319,15 +318,36 @@ def to_votable(self, limit: int | None = None) -> astropy.io.votable.tree.VOTabl
                 if doc := getattr(column, "doc"):  # noqa
                     extra_descriptions[name] = doc
 
-        votable = astropy.io.votable.tree.VOTableFile()
-        resource = astropy.io.votable.tree.Resource()
+        # Astropy <= 7.1 has a bug whereby it internally does not know its
+        # own version and so disallows some options. For now we have to use
+        # an internal API to assign the internal config. Otherwise certain
+        # elements can not be specified.
+        votable = astropy.io.votable.tree.VOTableFile(version="1.5")
+        votable_config = votable._get_version_checks()
```

Comment:
It's entirely possible that I'm missing an API but this is needed to allow TimeSys to be added. I'm not sure how this is even meant to work because you aren't even allowed to construct a `TimeSys` without passing in this dict even if astropy/astropy#18356 is fixed.

Reply:
When astrop/astropy#18356 is fixed things will be much simple because TableElement will automatically inherit the version config and `VOTableFile.config` will be able to be passed directly into the CooSys and TimeSys constructors.